doc/classes/DampedSpringJoint2D.xml

```diff
@@ -78,12 +78,18 @@
 	</methods>
 	<members>
 		<member name="damping" type="float" setter="set_damping" getter="get_damping">
+			The spring joint's damping ratio.
+			When the two bodies move into different directions the system tries to allign them to the spring axis again.
+			This value defines how fast this happens, a higher value makes the allignment happen faster.
 		</member>
 		<member name="length" type="float" setter="set_length" getter="get_length">
+			The spring joint's maximum length.
 		</member>
 		<member name="rest_length" type="float" setter="set_rest_length" getter="get_rest_length">
+			The spring joint's resting length. The joint will always try to go to back this length when pulled apart or pushed together.
```

Comment:
`When the bodies attached to the spring joint move they stretch or squash it. The joint always tries to resize towards this length. Default value: [code]add default[/code]`